dependencies64/ffmpeg/ffmpeg-build-linux/build.sh

```diff
@@ -448,3 +448,11 @@ CFLAGS="-I$TARGET_DIR/include -I$TARGET_DIR/usr/local/include" LDFLAGS="-L$TARGE
 	--enable-libv4l2 \
 	--enable-zlib
 make -j $jval && make install
+
```

Comment:
Keep it manual until there is a need for it to be automatic. Recompiling ffmpeg is not something that will be done on a regular basis, only after careful consideration.